proj/src/main/webapp/script.js

```diff
@@ -85,7 +85,8 @@ function appendMessage(payload){
     Realtime Database
  */
 
-const PATH = '/messages'; // can make this more detailed (for example add user ID)
+const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
+const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
```

Comment:
is this going to remain hardcoded? Also are all chats going to be under the /chat directory?

Reply:
the chat ID won't remain hardcoded but the rest of the path will. And yeah all the chats will be under /chat

Reply:
I think we can make `PATH` into a function like `getPath` that takes `CHAT_ID` as an input. 

Also I'm not sure if you want to store userId and stuff in the path because then it means you'd have to store all the messages once for each user that has access to them, unless there is a way to have multiple paths pointing at the same data with firebase.

Reply:
yeah userID won't be stored in the path, it will just be stored as metadata for each message